optimizely/decision/helpers_test.go

```diff
@@ -137,13 +137,15 @@ var testFeatRollout3334 = entities.Feature{
 
```

Comment:
In a separate PR I will move this over to a top-level test fixtures package or something so that we can use it in tests across other packages as well.